data/help/texts.json

```diff
@@ -305,19 +305,19 @@
+      "Q: It seems like everything I eat makes me sick!  What's wrong?\nA: Lots of the food found in towns is perishable and will only last a few days after the start of a new game.  The electricity went out several days ago, so fruit, milk and similar foods are the first to go bad.  After the first couple of days, you should switch to canned food, jerky, and hunting.  Also, you should make sure to cook any food and purify any water you come across, as it may contain parasites or otherwise be unsafe.",
       "Q: How can I remove boards from boarded-up windows and doors?\nA: Use a hammer and choose the direction of the boarded-up window or door to remove the boards.",
       "Q: The game just told me to quit, and other weird stuff is happening.\nA: You have the Schizophrenic trait, which might make the game seem buggy.",
-      "Q: How can I prevent monsters from attacking while I sleep?\nA: Find a safe place to sleep, for example in a cleared building far from the front door.  Set traps if you have them, or build a fire to scare off wild animals.",
+      "Q: How can I prevent monsters from attacking me while I sleep?\nA: Find a safe place to sleep; for example, in a cleared building far from the front door.  Set traps if you have them, or build a fire to scare off wild animals.",
       "Q: Why do I always sink when I try to swim?\nA: Your swimming ability is reduced greatly by the weight you are carrying, and is also adversely affected by the clothing you wear.  Until you reach a high level of the swimming skill, you'll need to drop your equipment and remove your clothing to swim, making it a last-ditch escape plan.  Diving gear may significantly help you in swimming and diving.",
       "Q: How can I cure a fungal infection?\nA: Royal jelly, the Blood Filter bionic, and some antifungal chemicals can cure fungal infection.  You can find royal jelly in the bee hives which dot forests.  Antifungal chemicals to cure the fungal infection can either be found as random loot or made from other ingredients.",
       "Q: How do I get into science labs?\nA: You can enter the front door if you have an ID card by examining (<press_examine>) the keypad.  If you are skilled in computers and have an electrohack, it is possible to hack the keypad.  An EMP blast has a chance to force the doors open, but it's more likely to break them.  You can also sneak in through the sewers sometimes, or try to smash through the walls with explosions.",
       "Q: Why does my crafting fail so often?\nA: Check the difficulty of the recipe, and the primary skill used; your skill level should be around one and a half times the difficulty to be confident that it will succeed.",
```

Comment:
```suggestion
      "Q: Why does my crafting fail so often?\nA: Check the difficulty of the recipe and the primary skill used; your skill level should be around one and a half times the difficulty to be confident that it will succeed.  Also note any proficiencies or other skills that may be useful.",
```

Reply:
Made a manual commit to address this suggestion